index.html

```diff
@@ -130,114 +130,146 @@ <h3 class="heading-secondary heading-light">Fine ingredients</h3>
     <section class="menu-section">
       <div class="menu-columns">
         <div>
-          <h4 class="heading-secondary heading-dark">Appetisers</h4>
-          <img src="gallery/decoration/decoration-2.png" class="decoration-menu" alt="decorative element">
-          <div class="menu-items">
-            <p>Tzatsiki</p>
-            <p>$3.99</p>
-          </div>
-          <div class="line-decoration"></div>
-          <blockquote class="description-of-menu-items menu-indent">Refreshing traditional cucumber and garlic yoghurt
-            dip.
-          </blockquote>
-          <div class="menu-items">
-            <p>Aubergine Salad</p>
-            <p>$5.50</p>
-          </div>
-          <div class="line-decoration"></div>
-          <blockquote class="description-of-menu-items menu-indent">Pureed eggplant, garlic, green pepper and tomato dip
-          </blockquote>
-          <div class="menu-items">
-            <p>Aubergine Salad</p>
-            <p>$5.25</p>
-          </div>
-          <div class="line-decoration"></div>
-          <blockquote class="description-of-menu-items menu-indent">Refreshing traditional cucumber and garlic yoghurt
-            dip.
-          </blockquote>
+          <article class="food-category appetisers">
+            <h4 class="heading-secondary heading-dark">Appetisers</h4>
+            <img src="gallery/decoration/decoration-2.png" class="decoration-menu" alt="decorative element">
+            <article class="dish and description">
+              <article class="menu-items">
+                <p>Tzatsiki</p>
+                <p class="price">$3.99</p>
+              </article>
+              <div class="line-decoration"></div>
+              <article class="description-of-menu-items">Refreshing traditional cucumber and garlic yoghurt
+                dip.
+              </article>
+            </article>
+            <article class="dish and description">
```

Comment:
Classes should refer to what you are using in the style sheet. They shouldn't be used as element descriptors. Consider simplifying the nested elements. If it has no styles and isn't helping with the layout, then it's probably not needed.